.github/workflows/ci_tests.yaml

```
@@ -41,20 +41,24 @@ jobs:
             isDraft: true
           - os: windows-latest
             isDraft: true
-          - os: ubuntu-latest
-            python-version: 3.7
-            isDraft: true
-          - os: ubuntu-latest
-            python-version: 3.8
-            isDraft: true
+          # - os: ubuntu-latest
+          #   python-version: 3.7
+          #   isDraft: true
```

For some reason, these lines can't be uncommented so that only one workflow (Ubuntu - Py3.9) runs. It fails with `
Error when evaluating 'runs-on' for job 'test'. (Line: 29, Col: 14): Unexpected value ''
` (see e.g. https://github.com/GenericMappingTools/pygmt/actions/runs/663292971), though that Line 29 appears totally unrelated.

```suggestion
          # - os: ubuntu-latest
          #   python-version: 3.7
          #   isDraft: true
```

Might have something to do with the 'include:' lines below? But maybe that's ok, we could just run 2 Linux tests (Python 3.7/NumPy 1.17 and Python 3.9/NumPy 1.20) in draft mode (i.e. +1 extra test compared to #906).

---

Yeah it's a tricky problem :sweat_smile: I'll leave it as it is and revise the MAINTENANCE.md docs to mention that two jobs are run instead of just one.